.gitattributes

```diff
@@ -0,0 +1,5 @@
+# Force the use of Unix LF style line endings
+* text eol=lf
```

Comment:
I am 99.94% sure this is not what we want.  On Windows CRLF is the native EOL, if you edit a file on Windows that is what the editor expects to see, and what it will (most likely) save.